manifests/cf-manifest/spec/jobs_spec.rb

```diff
@@ -0,0 +1,71 @@
+RSpec.describe "the global update block" do
+
+  let(:manifest) { manifest_with_defaults }
+
+  describe "in order to run parallel deployment by default" do
+    it "has serial false" do
+      expect(manifest["update"]["serial"]).to be false
+    end
+  end
+
+end
+
+RSpec.describe "the jobs definitions block" do
+
+  let(:jobs) { manifest_with_defaults["jobs"] }
+
+  def get_job(job_name)
+    jobs.select{ |j| j["name"] == job_name}.first
+  end
+
+  def is_serial(job_name)
+    job = get_job(job_name)
+    job["update"]["serial"]
+  end
+
+  def ordered(job1_name, job2_name)
+    i1 = jobs.index{ |j| j["name"] == job1_name }
+    i2 = jobs.index{ |j| j["name"] == job2_name }
+    i1 < i2
+  end
+
+  describe "in order to control jobs order relative to etcd_z1" do
```

Comment:
From meatspace: we'll duplicate this in each of the "in order to" blocks, because it belongs to both and doesn't currently describe why we care about `serial: true`.